activiti-api/activiti-task-model-api-impl/src/main/java/org/activiti/runtime/conf/TaskResultMixIn.java

```diff
@@ -0,0 +1,16 @@
+package org.activiti.runtime.conf;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.PROPERTY;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+
+@JsonTypeInfo(
+        use = NAME,
+        include = PROPERTY,
+        property = "resultType")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TaskResultMixIn {
```

Comment:
Same here, merge with `ProcessRsultMixIn` and move to common.

Reply:
@erdemedeiros look comment above.. I've lost too much time during the weekend on those mapping and now they are working.